include/pybind11/pytypes.h

```diff
@@ -1507,8 +1507,12 @@ class memoryview : public object {
         const void *ptr, ssize_t itemsize, const char *format,
         detail::any_container<ssize_t> shape,
         detail::any_container<ssize_t> strides) {
-        return memoryview::from_buffer(
-            const_cast<void*>(ptr), itemsize, format, shape, strides, true);
+        return memoryview::from_buffer(const_cast<void *>(ptr),
```

Comment:
Grr, this isn't the same as the other PR. I'll have to go and fix one of them. I prefer this style anyway though.

Reply:
Oh wait, it's caused by a bug. Fixed.